docs/learn/learn-nomination-pools.md

```diff
@@ -0,0 +1,143 @@
+Polkadot. They are designed to permissionlessly allow users (delegators) to pool their funds
+together and act as a single nominator account. Due to the current runtime constraints, Polkadot can
+only handle {{ max_active_nominator_count }} nominators comfortably in the
+[electing set](learn-nominator.md#staking-election-stages). As one of the objectives of the
+[NPoS algorithm](learn-phragmen.md) is to maximize the overall stake on the network, it can be
+inferred that the staking system on Polkadot favors nominators with a larger stake. Only the
+nomiator accounts which back the Validators in the active set are eligible for receiving staking
+rewards. This leaves out nomination intents from the accounts with lower DOT balance than the
+min-active nomination and places them in a waiting queue to enter electing set. Nomination pools
+will be handy to the users (delegators) who would like to participate the staking system with a
+stake much lower than the dynamic min-active nomination threshold on the network. All operations are
+constant space and time complexity relative to the number of delegators, eliminating any theoretical
+upper bound on the quantity of delegators the system can handle and thus scaling the number of
+accounts that can participate and earn rewards in the staking system on Polkadot.
+
+The earnings of the pool are split pro rata to a delegators stake in the bonded pool (and thus the
+staking rewards for delegators will be the same as if they were a nominator). Importantly, slashes
+are also applied proportionally to delegators who may have been actively bonded when a validator
+committed a slashable offence.
+
+## Key Components of Nomination Pools
+
+- Bonded Pool: Tracks the distribution of actively staked funds.
+- Reward Pool: Tracks rewards earned by actively staked funds.
+- Unbonding Sub Pools: Collection of pools at different phases (i.e. eras) of the unbonding
+  lifecycle.
+- Delegators: Accounts that are members of pools.
+- Point: A unit of measure for a delegator’s portion of a pool's funds.
+
+## Delegator Lifecycle
+
+### Join a pool
+
+A user delegates funds to a pool by transferring some amount to the pool’s bonded account with the
+join extrinsic. The pool then increases its bond with the new funds. A delegator is afforded the
+ability to bond additional funds and rewards as long as they are already actively bonded. Note that
+a delegator may only belong to one pool at a time.
+
+### Claim rewards
+
+The delegator can claim their portion of any rewards that have accumulated since the previous time
+they claimed (or in the case that they have never claimed, any rewards that have accumulated since
+the era after they joined). Rewards are split pro rata among the actively bonded delegators.
+
+### Unbond funds
+
+At any point in time after joining the pool, a delegator can start the process of exiting by
+unbonding. `unbond_other` will unbond all of the delegators funds. Once this call is made, the
```

Comment:
outdated name of transaction. 

To make sure you are not wrong, you should link to the substrate rustdocs, perhaps: https://paritytech.github.io/substrate/master/sc_service/index.html

Reply:
also, we have added partial unbonding in the meantime.

Reply:
also, you can have at most 16 partial unbonds, similar to staking.